docs/src/main/paradox/routing-dsl/index.md

```diff
@@ -57,8 +56,6 @@ Scala
 Java
```

Comment:
Did you keep the lines above intentionally?

Reply:
How could I not see this? Removed.